### PR TITLE
[4.10] Add STS Disconnect cluster jobs

### PR DESCRIPTION
**Pull request by yunjiang29**
A follow-up PR of #35241, as https://issues.redhat.com/browse/OCPBUGS-2743 has been fixed, adding `aws-ipi-disconnected-cco-manual-sts-ep` fpr amd&arm to:
* 4.10 nightly & stable
* upgrade: 4.10 stable to 4.11 nightly

### CODE REVIEW

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-aws-ipi-disconnected-cco-manual-sts-ep-p2-f14

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-4.11-upgrade-from-stable-4.10-aws-ipi-disconnected-cco-manual-sts-ep-p2-f7

**Comment by yunjiang29** [comment]
@jianlinliu PTAL

**Comment by yunjiang29** [comment]
install get passed:
* https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/36450/rehearse-36450-periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-aws-ipi-disconnected-cco-manual-sts-ep-p2-f14/1626509352665878528
* https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/36450/rehearse-36450-periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-4.11-upgrade-from-stable-4.10-aws-ipi-disconnected-cco-manual-sts-ep-p2-f7/1626509424665300992

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10.yaml [hunk 1]
```diff
@@ -80,8 +80,21 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
-- as: aws-ipi-disconnected-cco-manual-sts-p2-f14
-  cron: 17 21 8,23 * *
+- as: aws-ipi-disconnected-cco-manual-sts-ep-p2-f7
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10.yaml:83]: can we keep the original test frequency - f14?

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10.yaml:83]: ahh, you've decreased the frequency https://github.com/openshift/release/pull/36490,  I copied it from the previous version. Okay, will sync up with your changes.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10.yaml:83]: Yeah, even per our agreement, if the install workflow is covered in both arm and amd, we even can set it to f28 for 4.10/4.11. But maybe deserve a separate PR for that.

**Comment by yunjiang29** [comment]
@liangxia PTAL

**Pull request merged**